tests/fixtures/tests.py

```diff
@@ -917,14 +917,12 @@ def test_loaddata_error_message(self):
             management.call_command("loaddata", "invalid.json", verbosity=0)
 
     @unittest.skipUnless(
-        connection.vendor == "postgresql", "psycopg2 prohibits null characters in data."
+        connection.vendor == "postgresql",
+        "PostgreSQL prohibits null characters in data.",
```

Comment:
I would rather the message say "psycopg guards against null characters in data." since that's[ where this exception comes from](https://github.com/psycopg/psycopg/blob/5acb61464502dd7ca9ebd1c51b724ca472a4778d/psycopg/psycopg/types/string.py#L40). It might be better to clarify that `ValueError` is from psycopg and `DataError` from psycopg3 so that can be cleaned up later (or add an if statement to make it explicit). Further, if someone wanted to encapsulate the vendor check into a database feature, this test could work on CockroachDB which also uses psycopg.

Reply:
ACK, I will fixup the `ValueError/DataError` usage so we know what to drop once we remove support for `psycopg2`. While I agree that a database feature would be nice I am not sure how it would look like `throws_value_or_data_error_for_null_in_strings` seems a bit ugly… And then different databases might throw different errors etc…